bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gsaoi/Gsaoi.java

```diff
@@ -424,8 +424,6 @@ public static OdgwSize valueOf(String name, OdgwSize nvalue) {
 
     }
 
-    private static final String VERSION = "2011A-1";
```

Comment:
Is it right to remove the version?

Reply:
GSAOI seems to be missing the call to `setVersion` that the others do.  The version is written out to XML and used during migration to detect whether to apply updates to the XML before importing.  Instead of deleting the VERSION we should probably `setVersion` in the constructor like the other instrument data objects.

Reply:
Okay, I'll restore that. Is 2011A-1 indeed the correct version?

Reply:
It was picking up the default, 2009A-1, so we might as well use that.